faasmcli/faasmcli/tasks/__init__.py

```diff
@@ -23,6 +23,7 @@
 
 # Default names
 ns = Collection(
+    bare_metal,
```

Comment:
@Shillaker I guess we are missing the code for this task?

Reply:
What do you mean? The file is [here](https://github.com/faasm/faasm/blob/master/faasmcli/faasmcli/tasks/bare_metal.py). All the bare metal stuff is legacy and needs to be removed fully in a separate PR.

Reply:
Sorry, my point was that I did not understand why this file was modified (i.e. the `bare_metal` namespace added).